src/main/kotlin/org/rust/cargo/runconfig/RsExecutableRunner.kt

```diff
@@ -80,14 +81,15 @@ abstract class RsExecutableRunner(
         val runCargoCommand = state.prepareCommandLine()
         val workingDirectory = getWorkingDirectory(environment.project, runCargoCommand, artifact)
         val (_, executableArguments) = parseArgs(runCargoCommand.command, runCargoCommand.additionalArguments)
-        val runExecutable = createGeneralCommandLine(
+        val runExecutable = state.toolchain.createGeneralCommandLine(
             binaries.single().toPath(),
             workingDirectory,
             runCargoCommand.redirectInputFrom,
             runCargoCommand.backtraceMode,
             runCargoCommand.environmentVariables,
             executableArguments,
-            runCargoCommand.emulateTerminal
+            runCargoCommand.emulateTerminal,
+            patchToRemote = false
```

Comment:
Why don't we patch command line here?

Reply:
It's done for debugger/profiler/valgrind on CLion side if needed.